### ISSUE TITLE
Using GridStack w/ React.js causes items to overlap

### ISSUE THREAD

**Issue by Sawtaytoes** [troolee/gridstack.js#339]
For some reason, items are always overlapping when GridStack is used within a React.js project. I am using Webpack, but have no global vars defined. I only need jQuery on this one view.

Here's my module:
```jsx
import React, { Component } from 'react'
import $ from 'jquery'
import 'jquery-ui'
// import 'jquery-ui-touch-punch'
import 'gridstack/dist/gridstack'

// Utilities
import { stylesHelper } from 'utilities/styles-helper'

// Styles
const styles = [
    require('gridstack/dist/gridstack.css'),
    require('styl/gridstack-overrides'),
    require('styl/stats')
]

class Stats extends Component {
    renderGridStack() {
        $(function () {
            let options = {
                animate: true,
                cellHeight: 80,
                // float: true,
                verticalMargin: 10
            }

            setTimeout(() => {
                $('.grid-stack').gridstack(options)
            }, 0)
        })
    }

    render() { return (
        <article>
            <section className="bubble">
                <div className="grid-stack">
                    <div className="grid-stack-item"
                        data-gs-x="0" data-gs-y="2"
                        data-gs-width="4" data-gs-height="2"
                    >
                        <div className="grid-stack-item-content">
                            HI HI HI HI HI
                        </div>
                    </div>
                    
                    <div className="grid-stack-item"
                        data-gs-x="4" data-gs-y="0"
                        data-gs-width="4" data-gs-height="4"
                    >
                        <div className="grid-stack-item-content">
                            HI HI HI HI HI
                        </div>
                    </div>
                </div>
            </section>

            {this.renderGridStack()}
        </article>
    )}
}

export default stylesHelper(Stats, styles)
```

Why would it allow items to overlap each other?

**Comment by Sawtaytoes** [comment]
This is caused by npm using version 0.24-dev of Gridstack. It's fixed if I switch to 0.25-dev.

**Comment by radiolips** [comment]
In v0.2.4, `cellHeight` and `verticalMargin` should be `cell_height` and `vertical_margin`. Apologies for the inconvenience you've encountered. The options with underscores will continue to work in v0.2.4 and v0.3.0, but they will be removed in v1.0.0. Let me know if you're having a continued issue.
